validator/testdata/amp4ads_feature_tests/amp_story_ad_errors.out

```diff
@@ -40,6 +40,20 @@ amp4ads_feature_tests/amp_story_ad_errors.html:31:4 The mandatory attribute 'con
 >>     ^~~~~~~~~
 amp4ads_feature_tests/amp_story_ad_errors.html:34:4 The attribute 'content' in tag 'meta name=amp-cta-landing-page-type' is set to the invalid value 'GAME'. [DISALLOWED_HTML]
 |
+|      <!-- Should fail due to no `content=`. -->
+|      <meta name="amp4ads-vars-foo">
+>>     ^~~~~~~~~
+amp4ads_feature_tests/amp_story_ad_errors.html:37:4 The mandatory attribute 'content' is missing in tag 'meta name=amp4ads-vars-*'. [DISALLOWED_HTML]
+|
+|      <!-- Should fail due unrecognized `name` -->
+|      <meta name="amp4ads-vars" content="bar">
+>>     ^~~~~~~~~
+amp4ads_feature_tests/amp_story_ad_errors.html:40:4 The attribute 'name' in tag 'meta name= and content=' is set to the invalid value 'amp4ads-vars'. [DISALLOWED_HTML]
```

Comment:
Note that you end up with the same problem here. The error message is choosing the tagspec for 'meta name= and content=' to produce an error message, rather than the tagspec for 'meta name=amp4ads-vars-*'.

It's not so bad in this case, I suppose as it still reads sanely.

I would prefer to see a design where the `name` attribute value was a fixed set of strings or a single string, but I can live with this.

Reply:
I see what you mean, I agree this could be better but there are tradeoffs with the all in one approach. Thanks for the clarity.